Remove @babel/polyfill dependency

nlfurniss opened the pull request:
## Summary
`@babel/polyfill` was removed in #191 so is not longer needed as a dep

## Testing Done
`yarn run tests:all` passes locally

nicolo-ribaudo commented:
Where was `@babel/polyfill` loaded? You probably also need to update it there 🤔

rwjblue commented:
I think we already removed the usage (we have been queuing up for a major release).

Can you double check that @nlfurniss ?

nlfurniss commented:
> I think we already removed the usage (we have been queuing up for a major release).
> 
> Can you double check that @nlfurniss ?

Yup, which explains its absence 😝 . Updated the PR to simply remove it.

rwjblue commented:
@nlfurniss - Would you mind rebasing? I re-enabled CI so it _should_ run now..

nlfurniss commented:
> @nlfurniss - Would you mind rebasing? I re-enabled CI so it _should_ run now..

Pushed an empty commit and all the checks are ✅

rwjblue commented:
Thanks @nlfurniss!
